pom.xml

```diff
@@ -627,7 +627,7 @@
             <!-- stops the "uses" clauses being added to "Export-Package" manifest entry -->
             <_nouses>true</_nouses>
             <!-- Stop the JAVA_1_n_HOME variables from being treated as headers by Bnd -->
-            <_removeheaders>JAVA_1_3_HOME,JAVA_1_4_HOME,JAVA_1_5_HOME,JAVA_1_6_HOME,JAVA_1_7_HOME,JAVA_1_8_HOME,JAVA_1_9_HOME</_removeheaders>
+            <_removeheaders>JAVA_1_3_HOME,JAVA_1_4_HOME,JAVA_1_5_HOME,JAVA_1_6_HOME,JAVA_1_7_HOME,JAVA_1_8_HOME,JAVA_1_9_HOME,JAVA_1_10_HOME,JAVA_1_11_HOME,JAVA_1_12_HOME</_removeheaders>

```

Comment:
Where are these headers used?  There is no Java 1.9, 1.10 and so on. These symbols are incorrect.

Reply:
sorry for the mistake.
the comment above shall be in another pull request.